src/ocm/components/Router.tsx

```diff
@@ -7,19 +7,26 @@ import { FeatureGateContextProvider, FeatureListType } from '../../common';
 import { routeBasePath } from '../config';
 import { AssistedUILibVersion } from './ui';
 
-export const AssistedUiRouter: React.FC<{ allEnabledFeatures: FeatureListType }> = ({
-  allEnabledFeatures,
-}) => (
+export const AssistedUiRouter: React.FC<{
+  allEnabledFeatures: FeatureListType;
+  canSelectCpuArchitecture?: boolean;
+}> = ({ allEnabledFeatures, canSelectCpuArchitecture }) => (
   <Provider store={store}>
-    <Router features={allEnabledFeatures} />
+    <Router features={allEnabledFeatures} canSelectCpuArchitecture={canSelectCpuArchitecture} />
```

Comment:
Isn't this outdated?
IIRC we said yesterday that we are ditching this prop in favor of the features prop?

Reply:
Please, if there is more work to be done in this PR consider marking it as draft

Reply:
I'm working in day2